src/pybind/mgr/progress/module.py

```diff
@@ -327,6 +327,43 @@ def config_notify(self):
                     self.get_module_option(opt['name']))
             self.log.debug(' %s = %s', opt['name'], getattr(self, opt['name']))
 
+    def balancer_crush_compat(self, old_map, old_dump, new_map, msg):
+        # For the crush compat to use in creating a PgRecovery event
+        
+        osd_list = []
+        affected_pgs = []
+        for pool in old_dump['pools']:
+            pool_id = pool['pool']
+            for ps in range(0, pool['pg_num']):
+
+                # Was this OSD affected by the OSD coming in/out?
+                # Compare old and new osds using
+                # data from the json dump
+                old_up_acting = old_map.pg_to_up_acting_osds(pool['pool'], ps)
+                old_osds = set(old_up_acting['acting'])
+
+                new_up_acting = new_map.pg_to_up_acting_osds(pool['pool'], ps)
+                new_osds = set(new_up_acting['acting'])
+                osds_dif = old_osds - new_osds
```

Comment:
nit, s/dif/delta/

Reply:
@kamoltat this comment is not addressed.

Reply:
@tchaikov yep so I changed the var name of osds_dif to osds_delta. Is that okay?